### PR TITLE
supplement distribution

### PR DESCRIPTION
**Pull request by AlekhyaArasavalli**

### CODE REVIEW

File: Backend/routes/routes.py [hunk 1]
```diff
@@ -230,3 +230,75 @@ async def update_child_malnutrition(id: str, childs: ChildMalnutrition):
...
+# Delete a Supplementary detail
+@supp_router.delete('/supplementary/{supplementary_id}')
+def delete_supplementary(supplementary_id: str):
+    result = SupplementaryCollection.delete_one({'_id': ObjectId(supplementary_id)})
+    if result.deleted_count == 1:
+        return {'message': 'Supplementary detail deleted successfully'}
+    else:
+        raise HTTPException(status_code=404, detail='Supplementary detail not found')
+
+
```
Comment (mjayakumarmca) [inline Backend/routes/routes.py]: Could you please remove these extra lines

File: app/routes.py [hunk 2]
```diff
@@ -5,114 +5,7 @@
...
-        return {"error": "Post id not exist"}
-
-    for ngo in ngo_users:
-        if ngo["id"] == id:
-            print("ID matched")
-            return {"data": ngo}
-=======
-from app.model import User,Ngo
-from app.auth.jwt_handler import signJWT
-from app.auth.jwt_bearer import JWTBearer
```
Comment (mjayakumarmca) [inline app/routes.py:109]: have you removed all JWT related things?

**Review by mjayakumarmca — CHANGES_REQUESTED** [review]
Given couple of comments

File: app/routes.py (hunk 2 above)
Comment (AlekhyaArasavalli) [reply app/routes.py:109]: no...while it got merged shown me some conflicts that only i removed

File: Backend/routes/routes.py (hunk 1 above)
Comment (AlekhyaArasavalli) [reply Backend/routes/routes.py]: removed

**Review by mjayakumarmca — APPROVED** [review]
LGTM

**Pull request merged**